temba/channels/models.py

```diff
@@ -1428,9 +1432,16 @@ def send_junebug_message(cls, channel, msg, text):
 
         if is_ussd:
             session = USSDSession.objects.get_session_with_status_only(msg.session_id)
-            external_id = Msg.objects.values_list('external_id', flat=True).filter(pk=msg.response_to_id).first()
-            # NOTE: Only one of `to` or `reply_to` may be specified
-            payload['reply_to'] = external_id
+            # make sure USSD responses are only valid for a short window
+            response_window = timedelta(seconds=180)
```

Comment:
Might be slightly clearer to just calculate the oldest here instead of in the `if`. .

IE: 
```
response_expiration = timezone.now() - response_window
```